jekyll/_cci2/jobs-steps.md

```diff
@@ -33,89 +25,110 @@ The following diagram illustrates how data flows between jobs:
 
 ![Jobs Overview]( {{ site.baseurl }}/assets/img/docs/jobs-overview.png)
 
-Jobs can be run using the `machine` (linux), macOS or Windows executors, or the `docker` executor, which can compose Docker containers to run your jobs and any services they require, such as databases.
+**Note**: The jobs names shown in this diagram are just examples, you can call your jobs whatever you want.
```

Comment:
I know this is technically correct, but "jobs names" is just so awkward to read/say. Maybe "the names of the jobs"?

Reply:
I actually think that was a mistake on my part! Thanks for spotting